src/Components/ArtworkAlert/Components/Form/PriceRange.tsx

```diff
@@ -0,0 +1,41 @@
+import { Expandable, Spacer } from "@artsy/palette"
+import { FC } from "react"
+
+import { useArtworkAlertContext } from "Components/ArtworkAlert/Hooks/useArtworkAlertContext"
+import { PriceRange } from "Components/PriceRange/PriceRange"
+import {
+  CustomRange,
+  DEFAULT_PRICE_RANGE,
+} from "Components/PriceRange/constants"
+
+interface PricaRangeFilterProps {
+  expanded?: boolean
+}
+
+export const PriceRangeFilter: FC<PricaRangeFilterProps> = ({
+  expanded = true,
+}) => {
+  const { state, dispatch } = useArtworkAlertContext()
+
+  const handlePriceRangeUpdate = (updatedRange: CustomRange) => {
+    dispatch({
+      type: "SET_CRITERIA_ATTRIBUTE",
+      payload: { key: "priceRange", value: updatedRange.join("-") },
+    })
+  }
```

Comment:
**walkthrough**: Same as above. This file is almost identical to the one used in the existing flow with the exception of using the ArtworkAlertContext instead.